manifests/backup/backup-gcs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: pingcap.com/v1alpha1
+kind: Backup
+metadata:
+  name: demo1-backup-gcs
+  namespace: test1
+spec:
+  gcs:
+    projectNumber: your-project-id
+    secretName: gcs-secret
+  storageType: gcs
+  cluster: demo1
+  tidbSecretName: backup-demo1-tidb-secret
+  storageClassName: rook-ceph-block
```

Comment:
Why the backup requires `rook-ceph-block` storage class? For any PV we need, we should use the default or just standard which should work out of the box in most environments. For example, minikube, kind, gke and eks all have a standard storage class.

Reply:
Just for test，my test env use rook-ceph-block as PV provisioner, I will change it to local-storage

Reply:
I mean you should leave it empty or use `standard`. In most platforms, when the storage class name is empty, it will be set to the default storage class with admission controller.